libraries/AP_HAL_ChibiOS/hwdef/MambaF405US-I2C/hwdef.dat

```diff
@@ -0,0 +1,167 @@
+# hw definition file for processing by chibios_pins.py
+# Mamba Basic F405 US I2C
+# With F405 MCU, MPU6000 IMU and 7456 series OSD
+# Based on Mamba F405 from jeanphilippehell
+@ thanks to betaflight for pin information
+
+# MCU class and specific type.
+MCU STM32F4xx STM32F405xx
+
+HAL_CHIBIOS_ARCH_F405 1
```

Comment:
this should not be needed

Reply:
Removed the hal_chibios part, wouldn't compile without mcu